.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,29 @@
+name: "Test"
+
+on: push
```

Comment:
I think we generally want to run this in the context of a pull request, not on every push, right? So `on: pull_request`?